workflow/controller/operator.go

```diff
@@ -682,7 +682,8 @@ func (woc *wfOperationCtx) processNodeRetries(node *wfv1.NodeStatus, retryStrate
 
 		// See if we have waited past the deadline
 		if time.Now().Before(waitingDeadline) {
-			retryMessage := fmt.Sprintf("Retrying in %s", humanize.Duration(time.Until(waitingDeadline)))
+			woc.requeue(timeToWait)
```

Comment:
The manual requeue was not necessary before because the message update would immediately requeue the Workflow.

Since we avoid the message update, we must manually requeue it.